scriptharness/actions.py

```diff
@@ -6,8 +6,8 @@
 
 Attributes:
   LOGGER_NAME (str): logging.Logger name to use
-  STRINGS (dict): strings for actions.  In the future these may be in a
-    function to allow for localization.
+  STRINGS (Dict[str, Dict[str, str]]): strings for actions.  In the future
```

Comment:
This is accurate at the moment, but really, it's just a freeform dict of dicts and strings.  Are these docstrings really this inflexible?